src/_layouts/default.erb

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="<%= site.locale %>"  x-data="{ themeMode: window.localStorage.getItem('mode') || (window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light') }" :class="{'dark':themeMode=='dark','hidden':!themeMode}" class='hidden h-full' >
+<html lang="<%= site.locale %>"  x-data="{ themeMode: window.localStorage.getItem('mode') || (window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light') }" :class="{'dark':themeMode=='dark','hidden':!themeMode}":data-theme=themeMode class='hidden h-full' >
```

Comment:
you need to watch your spacing and quoting on changes like this.

Reply:
ok sorry sir i done mistake again due to word-wrap on in vs code